static_test_driver/static_test_driver.ino

```diff
@@ -207,7 +214,6 @@ void loop() {
       for (int i = 0; i < NUM_HIST_VALS; i++)
```

Comment:
When using the built-in tare() function, you can get rid of this manual calculation of a tare value